test/rtos_drivers/wifi/src/FreeRTOSIPConfig.h

```diff
@@ -46,19 +46,19 @@
 #endif
 
 /* User defined network parameters */
-#define IPconfig_IP_ADDR_OCTET_0    192
-#define IPconfig_IP_ADDR_OCTET_1    168
-#define IPconfig_IP_ADDR_OCTET_2    5
+#define IPconfig_IP_ADDR_OCTET_0    10
+#define IPconfig_IP_ADDR_OCTET_1    0
+#define IPconfig_IP_ADDR_OCTET_2    201
```

Comment:
A similar question for this change, does exposing this IP address make it easier for someone external to mount a denial-of-service attack?

Reply:
In this instance we are providing very minimal information to a secure internal wifi segment that has been specifically put in place with no routes to internal networks to prevent someone using it to access critical internal network data. 
That's to say, if some one does hack this network, then all they get is free internet access. 
Providing the Private A class address and net mask gives no real useful information about our secure actual internal layout, and as an attacker I would guess that we use either 10.x or 172.x. So while this limits the attack space (guessing), if you can crack the wifi password then you can sniff the traffic and IP's already, so it's a mute point really.
For DOD levels of security we would have these as pointers to a secure database as with the password, but we would also have the test wifi set up in a em shielded environment.

Reply:
Thanks for your answer, @cs-xm.